opentechinstitute/project-graphics
# Add vector versions of small, round commotion logo

Adds scalable version of the small, round commotion logo used at the top of the router ui.

To test:
* Make sure the images open properly and are editable
* Make sure the colors are correct.
* Make sure the proportions look right
* Make sure I put the logo in the right folder
